packages/manager/modules/ufrontend/url-builder.js

```diff
@@ -0,0 +1,36 @@
+import { Environment } from '@ovh-ux/manager-config';
+
+const buildQueryString = (data) =>
+  Object.keys(data)
+    .map((key) => `${encodeURIComponent(key)}=${encodeURIComponent(data[key])}`)
+    .join('&');
+
+export const buildUrl = (application, url, query) => {
```

Comment:
it's really nitpicking but buildURL would be more consistent (encodeURIComponent for example)